test/Language/TranslationCompletenessTest.php

```diff
@@ -91,7 +91,13 @@ public function testTranslations()
             }
         }
 
-        // If we have no extra and no missing translations, $err will be empty.
-        self::assertEmpty($err, $err);
+        // Make this test informational only.
+        if ($err !== '') {
+            $this->markTestSkipped("Translation completeness (informational):\n" . $err);
+            return;
+        }
+
+        // Avoid marking test as risky.
+        $this->addToAssertionCount(1);
```

Comment:
Looks like you are using a PHPUnit internal method here.

Why not just mark the test with the `@doesNotPerformAssertions` annotation ?